bin/stacks/routing-api-stack.ts

```diff
@@ -374,16 +381,25 @@ export class RoutingAPIStack extends cdk.Stack {
       const metric = new MathExpression({
         expression: '100*(response200/(invocations-response400))',
         usingMetrics: {
-          invocations: api.metric(`GET_QUOTE_REQUESTED_CHAINID: ${chainId.toString()}`, {
-            period: Duration.minutes(5),
+          invocations: new aws_cloudwatch.Metric({
+            namespace: 'Uniswap',
+            metricName: `GET_QUOTE_REQUESTED_CHAINID: ${chainId.toString()}`,
```

Comment:
so we already had per chain? why was it not working?

Reply:
Because it was incorrectly implemented. per-chain metrics are logged from routing lambda, but the implementation was trying to get the per-chain metrics from API gateway, so that the alarm always has insufficient datapoints to evaluate, resulting in no alarm across all time.

I fixed it here, and verified now it's working on my local.

Reply:
I think this is just the metric but not the alert, the alert is whats added on line 417. That's what I understand

Reply:
I originally thought this was just the metric, but I saw then that it is indeed the alert.

Reply:
nah 5 minutes evaluation period is fine